deployment/terraform/storage.tf

```diff
@@ -0,0 +1,18 @@
+resource "aws_s3_bucket" "logs" {
+  bucket = "${lower(var.project)}-${lower(var.environment)}-logs-${var.aws_region}"
+  acl    = "private"
+
+  tags {
+    Name        = "${lower(var.project)}-${lower(var.environment)}-logs-${var.aws_region}"
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+module "ecr-repository" {
+  source = "github.com/azavea/terraform-aws-ecr-repository?ref=0.1.0"
+
+  repository_name = "${lower(var.project)}"
```

Comment:
I feel as though this should match what we have in the Docker Compose file for CI so that it is easier to establish the connection between the two.

Reply:
There isn't anything defined for `OAR_AWS_ECR_ENDPOINT` in `docker-compose.ci.yml`. This is stored as a credential in Jenkins as to not reveal our AWS Account ID.

Reply:
That being said, maybe we could rename this to the string literal `openapparelregistry`?

Reply:
In `docker-compose.ci.yml`, we build the container image as `open-apparel-registry`. Here we are rendering it as `openapparelregistry`.

I think that it makes sense to make the repository name (minus the repository specific bits) consistent across the two so that it is easier to make the connection between them.